### DISCUSSION TITLE
Perhaps adding HEIF/HEIC photo file type support

### DISCUSSION THREAD

**Discussion by HHH0003** [Xanashi/Icaros#77] [Feature Requests]
Windows 11 new file explorer is having issues with generating thumbnails for advanced codec file types.  Since Windows has removed native support for HEVC and HEIF/HEIC files... requiring codecs from the Microsoft Store, Explorer is having issues generating thumbnails for these file types.

**Comment by Xanashi** [comment]
The upcoming release of Icaros will feature HEIF/HEIC support. 
Some format features are not supported yet, but most HEIC files will produce a thumbnail.

**Comment by Xanashi** [comment]
While it's not feature complete it should work now in #82
